### ISSUE TITLE
MySQLDump error RE: password needed

### ISSUE THREAD

**Issue by katmoody** [humanmade/backupwordpress#1048]
Reported in a couple threads, there is a MySQLDump error coming up:

```
Mysqldump: Warning: Using a password on the command line interface can be insecure.
ERROR 1045 (28000): Access denied for user 'database login'@'10.168.1.12' (using password: YES)
```

Waiting for further feedback on one ticket regarding whether the backup works or not, on at least one ticket it does actually work in spite of the error. 

wp.org: 
https://wordpress.org/support/topic/backup-seems-to-work-but-this-error-is-reported?replies=1

Maybe: https://wordpress.org/support/topic/access-denied-error-after-backupwordpress-version-35-update?replies=2

**Comment by willmot** [comment]
It should just fallback to the PHP version of Mysqldump.

**Comment by katmoody** [comment]
Is that the PDO: mysql?

**Comment by willmot** [comment]
yep 👍

**Comment by willmot** [comment]
This will be resolved with release 3.6
